src/types/layout.ts

```diff
@@ -0,0 +1,27 @@
+import { LPSAxisDir } from './lps';
+
+export type ViewType = '2D' | '3D';
+
+export enum LayoutDirection {
+  V = 'V',
+  H = 'H',
+}
+
+export interface ViewProps {
+  viewDirection: LPSAxisDir;
+  viewUp: LPSAxisDir;
+}
+
+export type LayoutView = {
+  objType: 'View';
+  viewType: 'View2D' | 'View3D';
+  id: string;
+  props: ViewProps;
```

Comment:
So to me these ( along with the `viewType` ) are properties of the view and don't really belong in the layout. The `items` in a layout should be `Array<Layout | string>` as in a layout or a viewID. Then in the view store rather than saving just a list of id, we should save these props and `viewType` along with the id. Thoughts?

Reply:
The latest commit contains the changes to move the view specs into the store. Let me know how that seems to you.

Reply:
Looks good to me!